src/scene_graph/camera_node.hpp

```diff
@@ -55,7 +72,8 @@ namespace wr
 			m_inverse_projection(),
 			m_view_projection(),
 			m_camera_cb(),
-			m_planes()
+			m_planes(),
+			m_ortho_res(OrthographicWidth(1280), OrthographicHeight(-720))
```

Comment:
You don't really benefit from this approach, just use a std::pair with 2 integers for the width would be better. make sure to still default initialize though. Also the height shouldn't be -. Apply the minus later maybe?